fuzzing/tools/launcher.py

```diff
@@ -0,0 +1,33 @@
+# Lint as: python3
+"""
+This is the launcher script to provide a uniform command line interface behind a number of arbitrary fuzzing engines.
+"""
+
+from absl import app
+from absl import flags
+from subprocess import call, TimeoutExpired, STDOUT
+import os
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_integer('timeout', 20, 'test timeout.', lower_bound=0)
+
+
+def main(argv):
+    if len(argv) < 2:
+        raise app.UsageError("Too few arguments")
+
+    ret_code = 0
+    try:
+        # Is this the right way to execute the runnable?
```

Comment:
> Is this the right way to execute the runnable?

Really good question. I guess `call` will also work here, but then you will need to manage the subprocess itself: decide how to interpret exit statuses, propagate output, etc.

Since this is a launcher tool, I think it is safe to simply cede control of the execution to the fuzzer binary. You can use the `os.exec*` system calls, which essentially replaces the Python executable with the command passed as its argument.

@asraa WDYT?

Reply:
Is there anyway to timeout the process when using `os.exec*`, or any parameter to control that when running the binary?

Reply:
Yes, libFuzzer provides the `-timeout` argument: https://llvm.org/docs/LibFuzzer.html#options

Reply:
Awesome! I was struggling with this for quite a while!